### PR TITLE
Fixes issue with adding QIT authentication if password has spaces.

### LINKED ISSUE #9750: QIT authentication fails if password contains spaces
### Describe the bug

When trying to set up QIT (see: PaJDYF-6n7-p2), I noticed that we pass the password string (which is set in an env file) in via the `common.sh` script without `"` quoting it. This means, if the password contains a space, it gets parsed as a new word rather than the same string, causing the script to fail.

### To Reproduce

1. On `develop`, use a password with a space in it in the `local.env` file for the `QIT_PASSWORD` variable (it doesn't need to be the correct password to see this error).
2. You should see an error similar to: `  No arguments expected for "partner:add" command, got "<password>".

### PR DESCRIPTION
**Pull request by dmallory42**
Fixes #9750 

#### Prerequisites

Please read the P2 post linked for context on QIT and create the `local.env` file as specified in the instructions.

#### Testing instructions

1. On `develop`, use a password with a space in it in the `local.env` file for the `QIT_PASSWORD` variable (it doesn't need to be the correct password to see this error).
2. Run the command `npm run test:qit-security`
3. You should see an error similar to: `  No arguments expected for "partner:add" command, got "<password>".`
4. Verify that this doesn't happen on this branch. (You can check that the steps on the p2 post linked on this page work as expected).

See: PaJDYF-6n7-p2 
-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `9751` or branch name `dev/fix-qit-spacing-error` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=dev/fix-qit-spacing-error) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 87e1c50e728e0786279583e50f71dd87fd19e051
- Build time: 2024-11-19 09:53:24 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: assets/css/admin.rtl.css [hunk 1]
```diff
@@ -155,11 +155,93 @@
 	padding-top: 2px;
 }
 
+/* WCPay Fraud Risk Level meta box */
```
Comment (oaratovskyi) [inline assets/css/admin.rtl.css:158]: @dmallory42 I wonder if these changed would be better to ship separately? they seem not related to the PR and there are no instructions where to test these design changes

Comment (dmallory42) [reply assets/css/admin.rtl.css:158]: I have no idea where these changes came from. 😅 Will try and remove them from the PR!

**Comment by dmallory42** [comment]
@oaratovskyi I had trouble getting this PR to play nicely so I just opened a new one: https://github.com/Automattic/woocommerce-payments/pull/9755

Closing this one!
